Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/CheckIn.kt

```diff
@@ -18,4 +18,29 @@ class CheckIn(
     val checkInEnd: Instant?,
     val targetCheckInEnd: Instant?,
     val createJournalEntry: Boolean
-)
+) {
+    @Suppress("ComplexMethod")
+    override fun equals(other: Any?): Boolean {
```

Comment:
why do you to override it ? I just ran the tests without it and PASSED

Reply:
removed it! thank you!